metric/system/process/helpers_windows.go

```diff
@@ -35,3 +38,24 @@ func isNonFatal(err error) bool {
 		errors.Is(err, syscall.EINVAL) ||
 		errors.Is(err, windows.ERROR_INVALID_PARAMETER) || errors.Is(err, NonFatalErr{})
 }
+
+func processesToIgnore() map[uint64]struct{} {
+	m := make(map[uint64]struct{})
+	// processesToIgnore checks if we should ignore the pid, to avoid elevated permissions
+
+	// LSASS.exe is a process which has no useful cmdline arguments, we should ignore acessing such process to avoid triggering Windows ASR rules
+	// we can query pid for LASASS.exe from registry
+	key, err := registry.OpenKey(registry.LOCAL_MACHINE, "SYSTEM\\CurrentControlSet\\Control\\Lsa", registry.READ)
+	if err != nil {
+		logp.L().Warnw("Failed to read registry path SYSTEM\\CurrentControlSet\\Control\\Lsa", "error", err)
+		return m
+	}
+	defer key.Close()
+	lsassPid, _, err := key.GetIntegerValue("LasPid")
```

Comment:
This would never have worked.  Can we add a unit test for this?

```suggestion
	lsassPid, _, err := key.GetIntegerValue("LsaPid")
```

Reply:
Ah, that’s why it wasn’t working. I double-checked everything but forgot about the key. I’m working on the unit test case and will update it with the correct key. Thanks for pointing that out!